chapters/packages.tex

```diff
@@ -227,12 +227,9 @@ \subsection{Directory Hierarchy Mapping}\label{mapping-a-package-class-hierarchy
 
 \subsection{Single File Mapping}\label{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}\label{single-file-mapping}
 
-When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), that file shall only define a single class \lstinline!A! with a name matching the name of the nonstructured entity.
-In a file hierarchy the files shall have the extension \filename{.mo}.
-
-A \filename{.mo} file defining more than one class cannot be part of the mapping
-to file-structure and it is an error if it is loaded from the
-\lstinline!MODELICAPATH!.
+When mapping a package or class-hierarchy to a file (e.g., the file \filename{A.mo}), the file content shall match \lstinline[language=grammar]!stored-definition! in the grammar.
+In this case, the \lstinline[language=grammar]!stored-definition! shall only define a single class with name (here, \lstinline!A!) matching the name of the nonstructured entity.
```

Comment:
```suggestion
In this case, the \lstinline[language=grammar]!stored-definition! shall only define a single class with a name (here, \lstinline!A!) matching the name of the nonstructured entity.
```
I think the indefinite article should be used here.

Reply:
@henrikt-ma do you agree with the suggested change?

Reply:
No, my non-English ears don't like the presence of _a_ here.  When _a_ is there, I get the impression that the class could have any number names, but without _a_ I get the impression that it is a given that every class has a name.

Reply:
This was settled in a later conversation.  Resolving.